### ISSUE TITLE
Writing file over USB hangs forever

### ISSUE THREAD

**Issue by Frijol** [tessel/t2-cli#233]
```sh
➜  test-community  t2 run gesture.js
INFO Connecting to Tessel...
INFO Connected to Kelsey over USB
INFO Writing gesture.js to RAM on Kelsey (2428.928 kB)...
```

been like this for at least five minutes.

Pushing this file:
```js
// example for the apds gesture sensor
var tessel = require('tessel');
var GestureLib = require('apds-gesture');
var G_THRESHOLD = 15
  , G_SENSITIVITY = 0.5//0.5
  ;

var gesture = GestureLib.use(tessel.port['A'],
    {'threshold': G_THRESHOLD, 'sensitivity': G_SENSITIVITY});

gesture.debug = true;

gesture.on('ready', function(){
  console.log("found a gesture sensor");
  gesture.setup(function(){
     gesture.readGesture();
  });
});

gesture.on('movement', function(dir){
  console.log("Sensed movement", dir);
});

gesture.on('error', function (err){
  console.log("Error: ", err);
});
```

**Comment by rwaldron** [comment]
Have you tried moving the commas to the end of the line?

**Comment by Frijol** [comment]
yep, no change

**Comment by rwaldron** [comment]
I think you just reverse-trolled me ;)

**Comment by dtex** [comment]
I'm seeing this as well. Hangs on a project that is a bit over 2MB.

**Comment by johnnyman727** [comment]
@dtex can you zip up your project and send it to me over Slack? That will help me repro.

**Comment by dtex** [comment]
It was a just a J5 blink. You probably shouldn't put much value into my comments until I've dug deeper. At the very least, wait until I can get the same error consistently.

**Comment by rwaldron** [comment]
@dtex are you working from the johnny-five repo or a fresh "project" directory with johnny-five npm installed?

**Comment by dtex** [comment]
Fresh project. Maybe I'm doing it wrong and need to get with my mentor sometime this week?

**Comment by rwaldron** [comment]
Hard to say, could be that we need to be more aggressive with .tesselignore. Let's do a hangout sometime this week

**Comment by johnnyman727** [comment]
Fixed with #486.
